### PR TITLE
Issue 2695: Remove newlines from GameOptions tooltips

### PR DESCRIPTION
**Pull request by SJuliez**
Partly solves issue #2695.

This removes most of the strangely placed newlines from the GameOptions tooltips and uses a sensible fixed width to make the tooltips break automatically.

image

instead of

image

### CODE REVIEW

File: megamek/i18n/megamek/common/options/messages.properties [hunk 1]
```diff
@@ -520,13 +520,13 @@ GameOptionsInfo.option.conditional_ejection.description=Only fire the ejection s
...
+GameOptionsInfo.option.begin_shutdown.description=Adds a deployment option to begin the game in a shutdown state. Requires Manual Shutdown/Startup
 GameOptionsInfo.option.max_external_heat.displayableName=(Unofficial) Maximum Heat from External Heat Sources
 GameOptionsInfo.option.max_external_heat.description=The maximum heat a unit takes per round from external heat sources (+15 is the official maximum).
 GameOptionsInfo.option.case_pilot_damage.displayableName=(Unofficial) CASE/CASEII reduces pilot damage on ammo explosion
 GameOptionsInfo.option.case_pilot_damage.description=If an ammo explosion occurs in a location protected by CASE/CASEII the pilot damage is reduced by 1.
 GameOptionsInfo.option.no_forced_primary_targets.displayableName=(Unofficial) Do not force primary target to be in front arc
-GameOptionsInfo.option.no_forced_primary_targets.description=If selected the Total Warfare rules on page 109\nthat require the primary target to be in the front arc are ignored
+GameOptionsInfo.option.no_forced_primary_targets.description=If selected the Total Warfare rules on page 109that require the primary target to be in the front arc are ignored
```
Comment (Windchild292) [inline megamek/i18n/megamek/common/options/messages.properties:529]:
```suggestion
GameOptionsInfo.option.no_forced_primary_targets.description=If selected the Total Warfare rules on page 109 that require the primary target to be in the front arc are ignored
```

File: megamek/i18n/megamek/common/options/messages.properties [hunk 2]
```diff
@@ -466,51 +466,51 @@ GameOptionsInfo.option.expanded_kf_drive_damage.description=If a K-F Drive criti
...
+GameOptionsInfo.option.pilot_advantages.description=If checked, players can add additional advantages to their pilots through the 'configure mech' window.
 GameOptionsInfo.option.edge.displayableName=Edge
 GameOptionsInfo.option.edge.description=if checked, pilots can be assigned edge which can trigger re-rolls in certain situations
 GameOptionsInfo.option.manei_domini.displayableName=Manei Domini Options
-GameOptionsInfo.option.manei_domini.description=VDNI,\nBuffered VDNI,\nand Pain Shunt as pilot options
+GameOptionsInfo.option.manei_domini.description=VDNI, Buffered VDNI, and Pain Shunt as pilot options
 GameOptionsInfo.option.individual_initiative.displayableName=(Unofficial) Individual Initiative
-GameOptionsInfo.option.individual_initiative.description=If checked,\neach unit has its own initiative roll,\nand must move in that fixed order.\n Good for zellbrigen or RPG style games.\nIncompatible with the lance movement options.
+GameOptionsInfo.option.individual_initiative.description=If checked, each unit has its own initiative roll, and must move in that fixed order.  Good for zellbrigen or RPG style games. Incompatible with the lance movement options.
```
Comment (Windchild292) [inline megamek/i18n/megamek/common/options/messages.properties:509]:
```suggestion
GameOptionsInfo.option.individual_initiative.description=If checked, each unit has its own initiative roll, and must move in that fixed order. Good for zellbrigen or RPG style games. Incompatible with the lance movement options.
```

File: megamek/i18n/megamek/common/options/messages.properties [hunk 3]
```diff
@@ -466,51 +466,51 @@ GameOptionsInfo.option.expanded_kf_drive_damage.description=If a K-F Drive criti
...
 GameOptionsInfo.option.front_load_initiative.displayableName=(Unofficial) Front load initiative
-GameOptionsInfo.option.front_load_initiative.description=If checked,\nthe team with the greater number of units moves them at the beginning of the initiative phase rather than the end,\ne.g.\nABBAB instead of ABABB as in the TW rules.
+GameOptionsInfo.option.front_load_initiative.description=If checked, the team with the greater number of units moves them at the beginning of the initiative phase rather than the end, e.g. ABBAB instead of ABABB as in the TW rules.
 GameOptionsInfo.option.initiative_streak_compensation.displayableName=(Unofficial) Initiative streak compensation
-GameOptionsInfo.option.initiative_streak_compensation.description=If checked,\nawards an increasing initiative roll bonus to a team that loses initiative several times in a row.\n Incompatible with Individual Initiative.
+GameOptionsInfo.option.initiative_streak_compensation.description=If checked, awards an increasing initiative roll bonus to a team that loses initiative several times in a row.  Incompatible with Individual Initiative.
```
Comment (Windchild292) [inline megamek/i18n/megamek/common/options/messages.properties:499]:
```suggestion
GameOptionsInfo.option.initiative_streak_compensation.description=If checked, awards an increasing initiative roll bonus to a team that loses initiative several times in a row. Incompatible with Individual Initiative.
```

File: megamek/i18n/megamek/common/options/messages.properties [hunk 4]
```diff
@@ -466,51 +466,51 @@ GameOptionsInfo.option.expanded_kf_drive_damage.description=If a K-F Drive criti
...
 GameOptionsInfo.option.protos_move_multi.displayableName=(Unofficial) move multiple ProtoMechs for every Mek or Vehicle
-GameOptionsInfo.option.protos_move_multi.description=If checked,\nmultiple ProtoMechs will have to move in place of a single Mek or Vehicle.\nSet the number per mech in the appropriate Game Option.\n If there are less than above specified protos remaining,\nthey all must move.\n The move order includes the presence of Protomechs.\nMutually exclusive with "Protomechs move after Meks and Vehicles".\nUnchecked by default.
+GameOptionsInfo.option.protos_move_multi.description=If checked, multiple ProtoMechs will have to move in place of a single Mek or Vehicle. Set the number per mech in the appropriate Game Option.  If there are less than above specified protos remaining, they all must move.  The move order includes the presence of Protomechs. Mutually exclusive with "Protomechs move after Meks and Vehicles". \nUnchecked by default.
```
Comment (Windchild292) [inline megamek/i18n/megamek/common/options/messages.properties:483]:
```suggestion
GameOptionsInfo.option.protos_move_multi.description=If checked, multiple ProtoMeks will have to move in place of a single Mek or Vehicle. Set the number per Mek in the appropriate Game Option. If there are less than above specified protos remaining, they all must move. The move order includes the presence of ProtoMeks. Mutually exclusive with "ProtoMeks move after Meks and Vehicles". Unchecked by default.
```

**Review by Windchild292 — CHANGES_REQUESTED** [review]
This replaces single spaced separations with double spaced separations, which is an accessibility issue. I've commented on four cases with problems before determining that's common in this PR.

**Comment by SJuliez** [comment]
Ok double spaces should be resolved. Also went through it all and replaced mech with Mek (also in Protomech and mechwarriors).

**Review by Windchild292 — APPROVED** [review]

**Pull request merged**